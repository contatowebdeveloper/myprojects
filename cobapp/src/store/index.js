import { createStore, applyMiddleware, compose } from 'redux'
import reducer from '../reducers'
import thunk from 'redux-thunk'

const logger = store => next => action => {
	//console.log('Action Type on Index.js:', action.type)
	//console.log('Redux Logger:', action)
	let result = next(action)
	//console.log('Next Logger:', store.getState())
	//console.log('Next Action Type on Index.js:', action.type)
	return result
}
const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose
export const store = createStore(
	reducer,
	composeEnhancers(
		applyMiddleware(thunk, logger)
	)
)