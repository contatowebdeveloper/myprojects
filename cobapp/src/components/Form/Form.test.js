import React from 'react'
import TestRenderer from 'react-test-renderer'

import { store } from '../../store'
import { Provider } from 'react-redux'

import FormDialog from './index'

describe('Render Modal component', () => {
    it('render correctly formDialog component', () => {
        const FormDialogComponent = TestRenderer.create(<Provider store={store} ><FormDialog /></Provider>).toJSON()
        expect(FormDialogComponent).toMatchSnapshot()
    })
})