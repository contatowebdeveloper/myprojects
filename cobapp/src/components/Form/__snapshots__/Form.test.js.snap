// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Modal component render correctly formDialog component 1`] = `null`;
